src/index.css

```diff
@@ -8,5 +8,51 @@
   font-family: 'Work Sans', sans-serif;
 }
 
+/* Global Styles */ 
 
+/* Ensures a parent element with this class name stretches to the height of floated children */
+.clearfix:after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+.container {
+  max-width: 1280px;
+  margin: 0 auto;
+}
+
+.primarySection {
+  padding: 120px 0;
+}
+
+.background2 {
+  background-color: gray;
+}
+
+.background3 {
+  background-color: #3b668e;
+  color: white;
+}
+
+.background4 {
+  background-color: white;
+}
+
+.float-left {
+  float: left;
```

Comment:
Your choice if you want to use this granular of helper classes. I just wanted to throw this here as an example. Float: left is something used often, and you may not always want to come up with a new class name for an element, if the only goal is to float it.